components/p3a/brave_p3a_service.cc

```diff
@@ -89,6 +90,7 @@ constexpr const char* kCollectedHistograms[] = {
     "Brave.Today.WeeklyMaxCardViewsCount",
     "Brave.Today.WeeklyMaxCardVisitsCount",
     "Brave.Sync.Status",
+    "Brave.Sync.ProgressTokenEverReset",
```

Comment:
can you also speficy in PR description what the new metrics are needed for?

Reply:
Updated PR description